sdk/go/common/resource/asset.go

```diff
@@ -1052,7 +1051,7 @@ func addNextFileToZIP(r ArchiveReader, zw *zip.Writer, seenFiles map[string]bool
 	// the ZIP format includes a date representation that starts at 1980. Use `SetModTime` to
 	// remain compatible with Go 1.9.
 	// nolint: megacheck
-	fh.SetModTime(time.Date(1990, time.January, 1, 0, 0, 0, 0, time.UTC))
+	fh.Modified = time.Date(1990, time.January, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
```suggestion
	fh.SetModTime(time.Date(1990, time.January, 1, 0, 0, 0, 0, time.UTC))
```
`SetModTime` handles more than just setting `Modified`

Reply:
I made this change due to [this issue](https://github.com/aonez/Keka/issues/422#issuecomment-508575155).

Reply:
![image](https://user-images.githubusercontent.com/22193008/194874677-32d33d63-7daa-4aef-891f-df6096aa58ac.png)
And I thought the package will automatically generate `ModifiedDate` and `ModifiedTime`.